Cypress 4.0 Docs (#1710)
cypress-io/cypress-documentation: source/api/cypress-api/isbrowser.md

title: Cypress.isBrowser
---

`Cypress.isBrowser` checks if the current browser matches the given name or filter.

# Syntax

```javascript
Cypress.isBrowser(name)
Cypress.isBrowser(filter)
```

## Arguments

**{% fa fa-angle-right %} name**  ***(String)***

The name of the browser (case-insensitive).

**{% fa fa-angle-right %} filter**  ***(Object)***

> Filter by the browser properties. You can inspect the current browser properties by using the {% url "Browser Launch API" browser-launch-api %}. Supported properties are:

Reviewer: can we link to https://docs.cypress.io/api/cypress-api/browser.html instead of the `before:browser:launch` docs here, since this is in the `driver`?

```suggestion
Filter by the browser properties. You can inspect the current browser's properties by looking at {% url "`Cypress.browser`" browser %}.
```

and then remove the list of properties below
